.github/workflows/build.yml

```diff
@@ -25,33 +25,33 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
 
-      - name: mvn verify
-        run: mvn verify
-
       - name: Installing dependencies
         run: mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 
+      - name: Verifying
+        run: mvn verify
```

Comment:
The install phase automatically includes verify. Is there any reason to call it again here?